.github/workflows/stale.yml

```diff
@@ -0,0 +1,45 @@
+name: Mark and close stale issues
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@5bef64f19d7facfb25b37b414482c7164d639639
+        with:
+          # Issues: mark after 60 days of inactivity, close after 14 more days
+          days-before-issue-stale: 60
+          days-before-issue-close: 14
```

Comment:
I think that this timing is way too aggressive. See the current distribution of ages in https://github.com/kubestellar/kubestellar/issues/3221#issuecomment-3180672023 (BTW, here is something else that could be contributed: a script that produces a CDF graph from the current live data on GitHub).

I suggest `days-before-issue-stale` should be 90 or 120 days, and `days-before-issue-close` be another 90 days.